docs/usage/api-usage.rst

```diff
@@ -0,0 +1,115 @@
+.. _api-usage:
+
+*********
+API Usage
+*********
+
+The **FAIR Data Point** exposes API endpoints that allow consumers to interact with the metadata. Some of the endpoints are available for all users, while others require an API token for authorization.
+
+Obtaining an API token
+======================
+
+In order to obtain an API token, you invoke the ``/tokens`` endpoint with your user credentials.
+
+.. code :: bash
+
+    curl -X POST -H "Accept: application/json" \
+        -H "Content-Type: application/json" \
+        -d '{ "email": "user@example.com", "password": "secret" }' \
+        https://fdp.example.com/tokens
+
+A successful call will return a ``JSON`` object with a token.
+
+.. code ::
+
+    { "token": "efIobn394nvJJFJ30..." }
+
+Issueing a request with the authorization token
+------------------------------------------------
+
+Subsequent requests should use this token in the `Authorization` header. The authorization type is a `Bearer <https://tools.ietf.org/html/rfc6750>`_ token.
+
+.. code :: bash
+
+    curl -H "Authorization: Bearer efIobn394nvJJFJ30..." \
+        -H "Accept: application/json" https://fdp.example.com/users
+
+Interacting with metadata
+=========================
+
+The metadata layers as defined by the :ref:`resource-definitions` are exposed through their respective endpoints. The general approach is that each layer, defined by its ``prefix``, supports a number of read and write HTTP methods.
+
+======== ====================== ========================
+Method   URL pattern            Functionality
+======== ====================== ========================
+``GET``  ``/${prefix}/${uuid}`` :ref:`retrieve-metadata`
+``POST`` ``/${prefix}``         :ref:`create-metadata`
+``PUT``  ``/${prefix}/${uuid}`` :ref:`update-metadata`
+======== ====================== ========================
+
+.. _retrieve-metadata:
+
+Retrieving metadata
+-------------------
+
+Retrieving metadata is open for GET requests without authorization.
```

Comment:
Just a minor thing, can you indicate that ur using dataset resources as an example here and the prefix for that resource is `dataset` ?